src/services/kratos/auth-phone-no-password.ts

```diff
@@ -141,6 +141,42 @@ export const AuthWithPhonePasswordlessService = (): IAuthWithPhonePasswordlessSe
     return kratosUserId
   }
 
+  const updatePhone = async ({
+    kratosUserId,
+    phone,
+  }: {
+    kratosUserId: UserId
+    phone: PhoneNumber
+  }) => {
+    let identity: Identity
+
+    try {
+      ;({ data: identity } = await kratosAdmin.getIdentity({ id: kratosUserId }))
+    } catch (err) {
+      return new UnknownKratosError(err)
+    }
+
+    if (identity.state === undefined)
+      throw new KratosError("state undefined, probably impossible state") // type issue
```

Comment:
We always return errors in application layer and below. We so far only throw in gql query resolvers.

We also always use curly braces if an if statement is more than 1 line https://blog.codecentric.de/curly-braces

```suggestion
    if (identity.state === undefined) {
      return new KratosError("state undefined, probably impossible state") // type issue
    }
```

Reply:
I just realized, this probably needs a more specific error type (new error type like `MissingStateForIdentityError`) and we can then leave out the message argument here

Reply:
That throw was copied over from the `upgradeToPhoneAndEmailSchema` method. I've updated it to a return now. I'm not adding another error class in this PR as this function will only be consumed via the admin panel. Will introduce a new error type in a follow up pr to fix it in `upgradeToPhoneAndEmailSchema` as well